workspaces/mta/rebuild-script.sh

```diff
@@ -19,12 +19,25 @@ yarn && yarn run tsc && yarn run build:all
 DYNAMIC_PLUGIN_ROOT_DIR=./deploy
 mkdir -p $DYNAMIC_PLUGIN_ROOT_DIR
 
-yarn --cwd plugins/mta-backend export-dynamic 
-yarn --cwd plugins/mta-frontend export-dynamic
-yarn --cwd plugins/catalog-backend-module-mta-entity-provider export-dynamic
-yarn --cwd plugins/scaffolder-backend-module-mta export-dynamic
-echo "Dynamic plugins exported"
+echo "Export dynamic plugins..."
+
+cd plugins/mta-backend
+npx -y @janus-idp/cli@^1.11.1 package export-dynamic-plugin --clean
+cd ../..
+
+cd plugins/mta-frontend
+npx -y @janus-idp/cli@^1.11.1 package export-dynamic-plugin --clean
+cd ../..
 
+cd plugins/catalog-backend-module-mta-entity-provider
+npx -y @janus-idp/cli@^1.13.0 package export-dynamic-plugin --no-embed-as-dependencies
+cd ../..
+
+cd plugins/scaffolder-backend-module-mta
+npx -y @janus-idp/cli@^1.13.0 package export-dynamic-plugin --no-embed-as-dependencies
```

Comment:
I used the versions from the package.json so that the script should work as before

Reply:
bit strange to see both 1.11.1 and 1.13.0 HARDCODED here -- how will renovate handle this?

Reply:
@ibolton336 does this look OK to you?

Reply:
I think it's a developer script; I just wanted to be kind and keep it working as it was before.

Renovate will not handle this. If you prefer I can remove the fixed version so that it uses always the latest version.

Reply:
Thanks for preserving this functionality for us. Changes look ok to me.

Reply:
I tried at least but I just run the script one and didn't tested the result on a cluster. Did you have done that? Or is anyway to merge this way?

Reply:
Testing now. Thx!